Themes/default/languages/ManageSettings.english.php

```diff
@@ -101,6 +101,22 @@
 // Like settings.
 $txt['enable_likes'] = 'Enable likes';
 
+// Privacy settings.
+$txt['enable_privacy_userexport'] = 'Enable Userdata Export by the User';
+$txt['privacy_userexport_own'] = 'Allow to Download ther own Data';
```

Comment:
ther -> their

Reply:
Your comment remind me,
to define the scope of this pr.

I will change this one thing,
but please look over the complet text after the pr is merge,
so you can check if the text make sense in the context where it's used.

Reply:
Sure, don't worry. I'll take a look at all the text strings once you're finished.